manifest_prod.yml

```diff
@@ -6,10 +6,23 @@ env:
   NEW_RELIC_CONFIG_FILE: newrelic.ini
   NEW_RELIC_ENV: production
 applications:
-- name: api
+- name: api-a
   path: .
   buildpack: "https://github.com/ddollar/heroku-buildpack-multi.git"
   host: fec-prod-api
+  services:
+    - fec-creds-prod
+  env:
+    PRODUCTION: True
+    NEW_RELIC_APP_NAME: OpenFEC API (production)
+    NEW_RELIC_LOG: stdout
+    WEB_CONCURRENCY: 4
```

Comment:
Can we also list somewhere here that we need the creds service to have (manually) set:

```
# NEW_RELIC_LICENSE_KEY
# SQLA_CONN
```